index.js

```diff
@@ -215,18 +221,26 @@ exports.toUnsigned = function (num) {
 }
 
 /**
- * Creates SHA-3 hash of the input
+ * Creates Keccak hash of the input
  * @param {Buffer|Array|String|Number} a the input data
- * @param {Number} [bits=256] the SHA width
+ * @param {Number} [bits=256] the Keccak width
  * @return {Buffer}
  */
-exports.sha3 = function (a, bits) {
+exports.keccak = function (a, bits) {
   a = exports.toBuffer(a)
   if (!bits) bits = 256
 
   return createKeccakHash('keccak' + bits).update(a).digest()
 }
 
+/**
+ * Creates SHA-3 (Keccak) hash of the input [OBSOLETE]
+ * @param {Buffer|Array|String|Number} a the input data
+ * @param {Number} [bits=256] the SHA-3 width
+ * @return {Buffer}
+ */
+exports.sha3 = exports.keccak
```

Comment:
Ok, just renaming to ``keccak`` since there is also the possibility to provide ``bits`` as parameter.